metadata.yaml

```diff
@@ -0,0 +1,17 @@
+name: apache-bigtop-zookeeper
```

Comment:
The name of the layer needs to be the name of the bigtop component, "hadoop-zookeeper".